tests/test_js_xpcom.py

```diff
@@ -221,32 +221,6 @@ def test_xpcom_shortcut_services_ww():
     """)
 
 
-def test_xpcom_nsibrowsersearchservice():
-    """
-    Test that currentEngine/defaultEngine are flagged when they are
-    overwritten.
-    """
-
-    assert not _do_test_raw("""
-    var foo = Cc["@mozilla.org/browser/search-service;1"]
-                   .getService(Components.interfaces.nsIBrowserSearchService);
-    alert(foo.currentEngine);
-    foo.bar = true;
-    """).failed()
-
-    assert not _do_test_raw("""
-    var foo = Cc["@mozilla.org/browser/search-service;1"]
-                   .getService(Components.interfaces.nsIBrowserSearchService);
-    foo.currentEngine = 123;
-    """).failed()
-
-    assert not _do_test_raw("""
-    var foo = Cc["@mozilla.org/browser/search-service;1"]
-                   .getService(Components.interfaces.nsIBrowserSearchService);
-    foo.defaultEngine = 123;
-    """).failed()
-
-
```

Comment:
On Tue, May 05, 2015 at 07:32:32AM -0700, Mathieu Agopian wrote:
>Is that linked to this PR? If not, please split those in a separate PR.

Yes. This tests exactly the opposite of what it's supposed to be 
testing, so one of my new checks causes it to fail.